app/controllers/users.js

```diff
@@ -25,7 +24,9 @@ const createUser = (firstName, lastName, email, password) =>
   }).catch(error => databaseError(error.message));
 
 exports.signUp = (req, res, next) => {
-  const { firstName, lastName, email, password } = req.body;
+  const { first_name, last_name, email, password } = req.body;
+  const firstName = first_name,
+    lastName = last_name;
```

Comment:
Change this in the next PR, but you can do `const { first_name: firstName, last_name: lastName, email, password } = req.body;`